dashboard/dashboard/pages/patients/__init__.py

```diff
@@ -324,8 +324,11 @@ def biground(x, base=5):
         )
         
 
-    fig.update_layout(barmode="overlay")
-    fig.update_traces(opacity=0.75)
+    fig.update_layout(barmode="overlay", bargap=0.1)
+    fig.update_traces(
+        opacity=0.75, 
+        hovertemplate='%{x}: %{y:.2f}<extra></extra>',
```

Comment:
What does that `<extra></extra>` part mean?

Reply:
without this you get an extra hover annotation which tells you the trace name e.g., "trace0". I didn't like how it looked so removed.
https://plotly.com/python/hover-text-and-formatting/#customizing-hover-text-with-a-hovertemplate